src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/AadPlugin.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.Utils;
+using Windows.Foundation.Metadata;
+using Windows.Security.Authentication.Web.Core;
+using Windows.Security.Credentials;
+
+namespace Microsoft.Identity.Client.Platforms.Features.WamBroker
+{
+    internal class AadPlugin : IWamPlugin
+    {
+        private readonly IWamProxy _wamProxy;
+        private readonly IWebAccountProviderFactory _webAccountProviderFactory;
+        private readonly ICoreLogger _logger;
+
+        public AadPlugin(IWamProxy wamProxy, IWebAccountProviderFactory webAccountProviderFactory, ICoreLogger logger)
+        {
+            _wamProxy = wamProxy;
+            _webAccountProviderFactory = webAccountProviderFactory;
+            _logger = logger;
+        }
+
+        public async Task<IEnumerable<IAccount>> GetAccountsAsync(string clientID)
+        {
+            var webAccounProvider = await _webAccountProviderFactory.GetAccountProviderAsync("organizations").ConfigureAwait(false);
```

Comment:
>"organizations" [](start = 93, length = 15)

nit: you have a constant value for this

Reply:
There are too many important constants in this implementation for them to be hidden away in some file. I'm gonna let them shine in the code :)